source/transactions/tests/retryable-commit-errorLabels.yml

```diff
@@ -55,10 +55,10 @@ tests:
             commitTransaction: 1
             lsid: session0
             txnNumber:
-            $numberLong: "1"
+              $numberLong: "1"
             startTransaction:
             autocommit: false
-            writeConcern: { w: majority, wtimeout: 10000 }
+            writeConcern:
```

Comment:
This is the only logic change I made, it doesn't get retried, so it shouldn't have a write concern.